rcl_logging_interface/CMakeLists.txt

```diff
@@ -32,6 +32,21 @@ install(DIRECTORY include/${PROJECT_NAME}/
   DESTINATION include/${PROJECT_NAME}
 )
 
+if(BUILD_TESTING)
```

Comment:
nit^N: It's more usual to see the following order

```cmake
ament_export_*(...)
...

if(BUILD_TESTING)
   ...
endif()

ament_package()
```

though there's no explicit rule about it

Reply:
done in b6013cbfb7292b11a160a770c578c00604bb70d9